src/js/utils/background.js

```diff
@@ -137,13 +182,28 @@ export const backgroundStyle = (backgroundArg, theme, textColorArg) => {
     theme,
   );
 
-  if (background.image) {
+  let backgroundImage = normalizeBackgroundImage(background, theme);
+
+  if (background.rotate) {
+    backgroundImage = rotateBackground(background, theme);
+  }
+
+  if (backgroundImage) {
     const backgroundStyles = `
       ${backgroundColor ? `background-color: ${backgroundColor};` : ''}
-      background-image: ${background.image};
+      background-image: ${backgroundImage};
       background-repeat: ${background.repeat || 'no-repeat'};
       background-position: ${background.position || 'center center'};
       background-size: ${background.size || 'cover'};
+      ${
+        background.clip && background.clip === 'text'
```

Comment:
Should be able to simply to `background.clip === 'text'`

Reply:
There are three possible values I want to return: 

- `-webkit-text-fill-color: transparent;
    -webkit-background-clip: text;
    background-clip: text;`
- `background-clip: ${background.clip};`
- `''` --> empty string when undefined

I will re-write to make more clear.

Reply:
There are three possible values I want to return:

- `
            -webkit-text-fill-color: transparent;
            -webkit-background-clip: text;
            background-clip: text;
          `
- `background-clip: ${background.clip};`
- `''` when undefined